sonic-ledd/scripts/ledd

```diff
@@ -124,10 +276,11 @@ def main():
 
     ledd = DaemonLedd()
 
-    # Listen indefinitely for changes to the PORT table in the Application DB's
+    # Listen indefinitely for port oper status changes
     while True:
```

Comment:
Should we move this while loop within the run routine ?
The way xcvrd/ycabled run change event monitoring loop.

Reply:
sure we can ...can this be done in next iteration?

Reply:
works